execution_workflows/APAtrap/samplesheet_example_files.csv

```diff
@@ -0,0 +1,3 @@
+condition,sample,bam,bai,bed
```

Comment:
Don't need `bed` column in header anymore

Reply:
oh yes thank you!